modules/simple-payments/paypal-express-checkout.js

```diff
@@ -161,14 +161,16 @@ var PaypalExpressCheckout = {
 								return reject( new Error( 'server_error' ) );
 							}
 
+							PaypalExpressCheckout.spayOrderId = paymentResponse.order_id;
```

Comment:
`PaypalExpressCheckout ` object is global for all payment buttons on a site.
Order id should be local to the transaction itself or at very lest the button.

To make it a local scope of the button, it should be a local variable in the `renderButton ` function.

ALSO we could make a global array that maps `paymentResponse.id` to `paymentResponse.order_id`. I think that would be safest.